src/filters/ext.rs

```diff
@@ -21,6 +21,17 @@ pub fn get<T: Clone + Send + Sync + 'static>(
     })
 }
 
+/// Get a previously set extension of the current route.
+///
+/// If the extension doesn't exist, it yields `None`.
+pub fn get_optional<T: Clone + Send + Sync + 'static>(
+) -> impl Filter<Extract = (Option<T>,), Error = Rejection> + Copy {
```

Comment:
Since this never rejects, it can change to `Error = Infallible`.

Reply:
Thanks. I'll fix it.